mocha-reporter-config.json

```diff
@@ -1,6 +1,6 @@
 {
   "reporterEnabled": "spec, mocha-junit-reporter",
   "mochaJunitReporterReporterOptions": {
-    "mochaFile": "./core-test-results/mocha/test-results.xml"
+    "mochaFile": "./test-results/mocha/test-results.xml"
```

Comment:
This file should be ignored in `.gitignore`.

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/1430/commits/b0acd8d30c093eac1d0d3417231a4367beab5edb